core/workflow-core/src/main/scala/edu/uci/ics/amber/core/storage/DocumentFactory.scala

```diff
@@ -50,9 +50,9 @@ object DocumentFactory {
         val storageKey = sanitizeURIPath(uri)
 
         val namespace = resourceType match {
-          case RESULT | MATERIALIZED_RESULT => StorageConfig.icebergTableResultNamespace
-          case CONSOLE_MESSAGES             => StorageConfig.icebergTableConsoleMessagesNamespace
-          case RUNTIME_STATISTICS           => StorageConfig.icebergTableRuntimeStatisticsNamespace
+          case RESULT             => StorageConfig.icebergTableResultNamespace
```

Comment:
we might need to differenticiate them. you decide.

Reply:
We cannot differentiate the two cases: if a port storage is both needed by the scheduler and indicated by the user, there should only be one storage, and consequently only one storageURI. The frontend has the ability to detect if a port storage is not specified by the user but scheduler-generated, so `MATERIALIZED_RESULT ` is not needed anyway.